editor/script/room.js

```diff
@@ -36,6 +36,8 @@ function RoomTool(canvas) {
 		var y = Math.floor( off.y / (tilesize*scale) );
 		// console.log(x + " " + y);
 
+		e.preventDefault();
```

Comment:
would it be possible to move the `preventDefault` call to the top of the function? it would be more consistent with the other fixes that way

Reply:
Ok, fixed